examples/timers/src/main.py

```diff
@@ -0,0 +1,107 @@
+from kybra import Duration, ic, nat8, query, Record, TimerId, update
+
+
+class StatusReport(Record):
+    single: bool
+    inline1: nat8
+    inline2: nat8
+    inner: str
```

Comment:
I don't understand why this inner thing is here when it's not in the JS example. Let's discuss

Reply:
Removed